client/src/app/routes/Landing.scss

```diff
@@ -289,6 +306,11 @@
                     color: var(--header-color);
                     font-family: $inter;
                     font-weight: 400;
+                    &.done-button {
+                        background: #14cabf;
```

Comment:
Change background color from hardcoded value to variable $mint-green-6.
Also possible solution is to add a theme variable if background color is supposed to change based on selected theme.